packages/datepickers/src/elements/Datepicker/Datepicker.tsx

```diff
@@ -30,66 +30,42 @@ import { DatepickerContext } from './utils/useDatepickerContext';
 import { StyledMenu, StyledMenuWrapper } from '../../styled';
 
 export interface IDatepickerProps {
-  /**
-   * The selected date to display
-   */
+  /** Sets the selected date */
   value?: Date;
   /**
-   * Returns the parsed date
+   * Handles changes to the selected date
+   *
+   * @param {Date} date The selected date to use
    */
   onChange?: (date: Date) => void;
-  /**
-   * Allows customization of date formatting within input.
-   */
+  /** Formats the date */
   formatDate?: (date: Date) => string;
-  /**
-   * Accepts [all valid Intl locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation)
-   */
+  /** Sets the locale. This accepts [all valid international locales](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation). */
   locale?: string;
-  /**
-   * Minimum value. Users are still able to manually enter a date below this value.
-   */
+  /** Sets the minimum value, but users can still enter a date below this value. */
   minValue?: Date;
-  /**
-   * Maximum value. Users are still able to manually enter a date above this value.
-   */
+  /** Sets the maximum value, but users can still enter a date above this value. */
   maxValue?: Date;
-  /**
-   * Show compact styling
-   */
+  /** Determines if compact styling is uesed */
   isCompact?: boolean;
   /**
-   * Override default date parsing. Receives a localized input value and returns a `Date` object.
+   * Parses the date with a custom format and returns a `Date` object
+   *
+   * @param {string} inputValue The format used for parsing
+   * @return {Date}
```

Comment:
Please prefer the default `@returns` tag rather than the `@return` synonym. https://jsdoc.app/tags-returns.html.

Reply:
Fixed in latest push.